src/ButtonStyles.js

```diff
@@ -4,83 +4,32 @@ import {get} from './constants'
 export default css`
   position: relative;
   display: inline-block;
-  padding: 6px 12px;
-  color: ${get('colors.gray.9')};
-  background-color: ${get('colors.gray.1')};
-  background-image: linear-gradient(-180deg, ${get('colors.gray.0')} 0%, ${get('colors.button.bg2')} 90%);
-  font-size: ${get('fontSizes.1')};
+  padding: 6px 16px;
   font-weight: ${get('fontWeights.bold')};
   line-height: 20px;
   white-space: nowrap;
   vertical-align: middle;
   cursor: pointer;
   user-select: none;
-  background-repeat: repeat-x;
-  background-position: -1px -1px;
-  background-size: 110% 110%;
-  border: 1px solid ${get('colors.button.border')};
```

Comment:
removing this and setting in each individual Button component since it's different for each component